webapp/package.json

```diff
@@ -51,7 +54,7 @@
     "eslint-plugin-react": "7.14.2",
     "file-loader": "4.0.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "24.9.0",
+    "jest": "^25.1.0",
```

Comment:
I wonder how these non-exact dependencies got installed with the `.npmrc` present. I wonder if there is a way we can configure a linter for this.

Reply:
This was from `npm audit fix` -- so maybe it doesn't honour `.npmrc`? I've fixed it manually for now.